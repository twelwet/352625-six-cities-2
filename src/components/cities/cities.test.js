import React from "react";
import renderer from "react-test-renderer";
import Cities from "./cities.jsx";

const citiesList = [`Moscow`, `Saint Petersburg`, `Helsinki`];

it(`Cities correctly renders after relaunch`, () => {
  const onCityClick = jest.fn();
  const onSelect = jest.fn();

  const tree = renderer
    .create(<Cities
      citiesList={citiesList}
      onCityClick={onCityClick}
      active={null}
      onSelect={onSelect}
    />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
