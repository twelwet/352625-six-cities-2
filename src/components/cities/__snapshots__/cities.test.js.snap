// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cities correctly renders after relaunch 1`] = `
<section
  className="locations container"
>
  <ul
    className="locations__list tabs__list"
  >
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item"
        href="#"
        onClick={[Function]}
      >
        <span>
          Moscow
        </span>
      </a>
    </li>
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item"
        href="#"
        onClick={[Function]}
      >
        <span>
          Saint Petersburg
        </span>
      </a>
    </li>
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item"
        href="#"
        onClick={[Function]}
      >
        <span>
          Helsinki
        </span>
      </a>
    </li>
  </ul>
</section>
`;
